j1939/README.md

```diff
@@ -0,0 +1,11 @@
+### SAE J1939 Tests
+
+J1939 tests are collection of scripts which was written to reproduce
+one or another bug.
+
+### Requirements
+
+- jcat and jacd should be installed on the DUT
```

Comment:
These tools are now named `j1939cat` and `j1939acd`. Please adjust the scripts and docs accordingly. The other comments are just nitpicks.